api-schema/src/main/java/org/stellar/anchor/api/sep/operation/Sep38Info.java

```diff
@@ -16,12 +16,6 @@ public class Sep38Info {
 
   Integer decimals;
 
-  @SerializedName("sell_delivery_methods")
```

Comment:
How does this class relate to `org.stellar.anchor.api.sep.sep38.InfoResponse`? It looks like we are going to be returning that instead.

Reply:
`InfoResponse` is a list of asset mapped from `Sep38Info`